### PR TITLE
Rewrite and overhaul Abteilungen page

### PR DESCRIPTION
**Pull request by LeonTReal**
TODO before merge:

- [x] Proof read the page

### CODE REVIEW

File: src/content/ueber-uns/abteilungen.svx
Comment (pantoffelemmy) [inline src/content/ueber-uns/abteilungen.svx:9]: This line should be removed before merging. I think you could use a HTML comment if you want to keep that line in the file for longer,

```suggestion
<!-- (TODO: Insert tfhc structure diagram here?) -->
```

I believe that would not be rendered.

File: src/content/ueber-uns/abteilungen.svx
Comment (pantoffelemmy) [inline src/content/ueber-uns/abteilungen.svx:17]: I'd suggest using strikethrough to mark dead links, looks a little bit cleaner imo.

```suggestion
_Bemerkung_: Infanterie, die aufgesessen auf [~~geschützten Fahrzeugen~~](../fuhrpark/geschuetzte-fahrzeuge) agiert, nennt man **motorisierte Infanterie**.
```

You could use searching for `~~` in your editor to find links that need curating later.

File: src/content/ueber-uns/abteilungen.svx
Comment (pantoffelemmy) [inline src/content/ueber-uns/abteilungen.svx:23]: I'm noticing some repeating sentence structure, maybe consider using a table to convey the information?

File: src/content/ueber-uns/abteilungen.svx
Comment (pantoffelemmy) [inline src/content/ueber-uns/abteilungen.svx:41]:
```suggestion
Die Truppengattung des Sanitätsdienstes wird von Emmi organisiert und verwaltet. Sie befasst sich mit der medizinischen Behandlung, Evakuierung und Endversorgung verwundeter Soldaten. Überwiegend sind Mitglieder des Sanitätsdienstes in die Infanterie eingegliedert, allerdings werden sie auch bei speziellen Unterstützungselementen wie dem beweglichen Arzttrupp (kurz BAT) oder dem MEDEVAC (<u>med</u>ical <u>evac</u>uation) eingesetzt. Der [Sanitäter](../sanitaetsdienst/sanitaeter/) ist die vielfältige Rolle des Sanitätsdienstes.
```

File: src/content/ueber-uns/abteilungen.svx
Comment (pantoffelemmy) [inline src/content/ueber-uns/abteilungen.svx:49]:
```suggestion
Die Truppengattung der Aufklärungstruppe wird von ? verwaltet und organisiert. Ihre Hauptaufgabe ist die Feldaufklärung feindlicher Truppenbewegungen während und außerhalb des Gefechts. Darüber hinaus werden Artilleriefeuer und Luftnahunterstützung durch die Aufklärungstruppe koordiniert und organisiert. Die Aufklärungstruppe umfasst die Rollen des [Scharfschützen](../aufklaerer/scharfschuetze), des [Beobachters](../aufklaerer/spotter) und des [JTAC](../aufklaerer/jtac).
```

**Review by pantoffelemmy — CHANGES_REQUESTED** [review]
I completely approve the wording itself and the new structure, while the latter is pretty fine grained it makes much sense.

I've left some formatting suggestions and minor corrections. I think this article could benefit further from the use of tables, but it reads well nonetheless. 

Thanks for your work.

File: src/content/ueber-uns/abteilungen.svx
Comment (LeonTReal) [reply src/content/ueber-uns/abteilungen.svx:9]: I do not intend to keep that line for long. Since this PR will have to wait for merge until https://github.com/taskforcehellcat/wiki/pull/63 is merged, I'm hoping for the new structure diagram to be finished by then. If not, we can simply remove this line before merge and insert the diagram later on.

File: src/content/ueber-uns/abteilungen.svx
Comment (LeonTReal) [reply src/content/ueber-uns/abteilungen.svx]: Resolved as of 48e52552c4ec8c8c2f617f947eaac8b529de6891.

File: src/content/ueber-uns/abteilungen.svx
Comment (LeonTReal) [reply src/content/ueber-uns/abteilungen.svx]: Not quite sure how. It's quite a bit of text for a table.

File: src/content/ueber-uns/abteilungen.svx
Comment (LeonTReal) [reply src/content/ueber-uns/abteilungen.svx]: Resolved as of 6b6a2463548a383b0ad312c6399fb1a78386040b.

File: src/content/ueber-uns/abteilungen.svx
Comment (LeonTReal) [reply src/content/ueber-uns/abteilungen.svx]: I simply used the acronyms definition as done by [Wikipedia](https://de.wikipedia.org/wiki/MedEvac).

Committed your suggested changes as of 706c576e1e342c2d6765a8d7dae7e9a045577a49.

File: src/content/ueber-uns/abteilungen.svx
Comment (LeonTReal) [reply src/content/ueber-uns/abteilungen.svx]: This was done on purpose. The standard NATO term for this is "Forward Air Controller" (FAC). The term "Joint Terminal Attack Controller" is [primarily being used by the United States Armed Forces](https://en.wikipedia.org/wiki/Forward_air_control#NATO). Since we do not roleplay US Forces, I found it more sensible to use the standard NATO term for this.

@pantoffelemmy @Venrix Feel free to comment your opinion on this.

Comment (pantoffelemmy) [reply src/content/ueber-uns/abteilungen.svx]: If FAC is nato terminology I'm for it. Let's just also change the JTAC article accordingly.

**Review by pantoffelemmy — APPROVED** [review]

File: src/content/ueber-uns/abteilungen.svx
Comment (pantoffelemmy) [reply src/content/ueber-uns/abteilungen.svx]: We can keep that in the back of our heads for later.

**Comment by pantoffelemmy** [comment]
@LeonTReal We might want to change the base of this PR from `main` to `rework-structure` if it depends on it.

**Comment by LeonTReal** [comment]
@pantoffelemmy Sure thing, go ahead if you want to.

**Pull request merged**
